actors/builtin/miner/expiration_queue.go

```diff
@@ -368,8 +368,35 @@ func (q ExpirationQueue) RemoveSectors(sectors []*SectorOnChainInfo, faults *abi
 		return nil, NewPowerPairZero(), xerrors.Errorf("failed to expand recoveries: %w", err)
 	}
 
+	// results
 	removed := NewExpirationSetEmpty()
 	recoveringPower := NewPowerPairZero()
+
+	// Process non-faulty sectors. These sectors can appear anywhere in the
+	// queue, but we know where to find them.
```

Comment:
This comment is a bit confusing now, given that active sectors are processed first.